addons/website_slides/models/slide_channel.py

```diff
@@ -161,6 +161,8 @@ def _default_access_token(self):
     karma_slide_comment = fields.Integer('Add a comment', default=3, help="Karma needed to add a comment on a slide of this course")
     karma_slide_vote = fields.Integer('Vote on slide', default=3, help="Karma needed to like/dislike a slide of this course.")
     can_review = fields.Boolean('Can Review', compute='_compute_karma_rights')
+    can_comment = fields.Boolean('Can Comment', compute='_compute_karma_rights')
```

Comment:
While we are at it I would rename compute karma rights to something like compute action rights ?